pkg/provider/provider.go

```diff
@@ -223,6 +223,15 @@ func buildTestEnvironment() { //nolint:funlen
 	operators := createOperators(data.Csvs, data.Subscriptions, data.AllInstallPlans, data.AllCatalogSources, false, false, true)
 	env.Operators = operators
 	logrus.Infof("Operators found: %d", len(env.Operators))
+	for _, pod := range env.Pods {
+		isCreatedByDeploymentConfig, err := pod.CreatedByDeploymentConfig()
+		if err != nil {
+			logrus.Warnf("Pod %s, err: %s", pod.String(), err)
+		}
+		if !isCreatedByDeploymentConfig {
```

Comment:
As I said in my previous comment, this value (isCreatedByDeploymentConfig) doesn't make any sense when err != nil, because the error means that you weren't able to determine whether it was created by a deploymentConfig or not, so just reporting the Warnf should be enough. You can add a "continue" in the `if err != nil`:
```
for _, pod := range env.Pods {
		isCreatedByDeploymentConfig, err := pod.CreatedByDeploymentConfig()
		if err != nil {
			logrus.Errorf("...")
			continue
		}
		if !isCreatedByDeploymentConfig {
		   logrus.Warnf("...")
		}
}
```